mobile_app1/benchmark.sh

```diff
@@ -6,11 +6,11 @@ if ! [ -x "$(command -v bazel)" ]; then
 fi
 
 if [[ -z "$GRADLE_PROFILER" ]]; then
-  if ! [ -d "gradle-profiler" ]
+  if ! [ -d "../gradle-profiler" ]
```

Comment:
Wouldn't it be better to move `benchmark.sh` to the toplevel directory?

Reply:
Yeah, that might make more sense. @oliviernotteghem , what do you think?

Reply:
Yeah, we can possibly move it up. It will require a little modifications to take project as input : currently we only have 1 mobile auto-gen project, but we can possibly generate one of each of Uber PROD mobile app.